syncano/connection.py

```diff
@@ -166,10 +166,7 @@ def build_params(self, params):
                 'X-API-KEY': self.api_key
             })
         elif self.api_key and 'Authorization' not in params['headers']:
-            if self.is_social:
-                params['headers']['Authorization'] = 'token %s' % self.token
-            else:
-                params['headers']['Authorization'] = 'ApiKey %s' % self.api_key
+            params['headers']['Authorization'] = 'token %s' % (self.token if self.is_social else self.api_key)
```

Comment:
This will need some further rework. 
Social is working kind of differently now (as user social wasn't working at all before):

* Admin can login by either login+password (`/v1/auth/`) or using `/v1/auth/facebook/` `/v1/auth/github/` `/v1/auth/google-oauth2/` and pass there access_token (as POST data) from his social service. In both cases, he will get as a return response with 'account_key' which should be use as either: `Authorization: token xxx` or `X-Api-Key: xxx` header for further communication (and connection can be initialized with those tokens skipping the login/password part which will probably be pretty common).

* Users can login in a similar fashion. Using instance api key + user login/password through `/v1/instance/xxx/user/auth/` or social with access_token as POST `/v1/instance/xxx/user/facebook/` `/v1/instance/xxx/user/google-oauth2/`. That will return `user_key` which should be used in combination with previously used instance api key, e.g. `Authorization: token xxx` + `X-User-Key: yyy` or `X-Api-Key: xxx` + `X-User-Key: yyy`.

So anyway, in none of these cases passing a token to Authorization header is used. It was meant to be used for users but that wasn't working at any point of time anyway.